src/main/java/com/futurcollars/lesson9/zad3/ImprovedCalculator.java

```diff
@@ -0,0 +1,32 @@
+package com.futurcollars.lesson9.zad3;
+
+public class ImprovedCalculator {
+
+    private int number1;
+    private int number2;
+
```

Comment:
Tworzenie kalkulatora tylko dla dwóch liczb jest bez sensu, to tak jakbyś musiał kupować nowy kalkulator za każdym razem jak chcesz użyć go na innych liczbach :p liczby do działań zawsze powinny przychodzić jako argumenty a sam kalkulator być bezstanowy :)